Recrutify/Recrutify.Services/Services/ScheduleService.cs

```diff
@@ -47,5 +56,25 @@ public async Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync(DateTime? date
             var schedules = await _scheduleRepository.GetByDatePeriodAsync(userId, date.Value, daysNum);
             return _mapper.Map<ScheduleDTO>(schedules);
         }
+
+        public async Task UpdateScheduleSlotsForCurrentUserAsync(IEnumerable<DateTime> dates)
+        {
+            var currentUserId = _userProvider.GetUserId();
+            var scheduleSlotByCurrentUser = await _scheduleRepository.GetScheduleSlotsByDatePeriodAsync(currentUserId, dates.Min());
```

Comment:
Please rename all vars on code according to count. Write s , where more than one. And rename all remote to removed